pdc/apps/package/serializers.py

```diff
@@ -139,6 +140,16 @@ def to_internal_value(self, data):
                                               rpm.id,
                                               'null',
                                               json.dumps(rpm.export()))
+
+                    if request and hasattr(request, '_messagings'):
+                        msg = {
+                            'url': reverse('rpms-detail', args=[rpm.pk], request=request),
+                            'author': request.user.username,
+                            'comment': request.META.get("HTTP_PDC_CHANGE_COMMENT", None),
```

Comment:
Aren't these fields always overriden by `NotificationMixin`?

Reply:
No, the mixin is adding another message. Once the message is in the queue, there is nothing that would change it's content. But it might be a good idea do some post processing somewhere.

Reply:
Oh, OK, got it. Would it be possible to move the common code for adding these fields to a helper function?